src/multivariate/mvtdist.jl

```diff
@@ -30,6 +59,8 @@ function GenericMvTDist(df::Real, Σ::AbstractPDMat)
     GenericMvTDist(df, Zeros{R}(size(Σ, 1)), Σ)
 end
 
+GenericMvTDist(df::Real, μ::AbstractVector, Σ::AbstractMatrix) = GenericMvTDist(df, μ, PDMat(Σ))
```

Comment:
New constructor added so no longer using the `mvtdist` ones which should be removed.

Reply:
That's a bit suboptimal as it will create a dense representation (in `PDMat`) also for `UniformScaling`, `Diagonal` matrices, etc. that should be converted to `ScalMat` and `PDiagMat`. I guess we want the same logic here as in e.g. `MvNormal` that creates these optimized `AbstractPDMat` types (and probably in the long run this should be supported by PDMats e.g. via a `AbstractPDMat` constructor).

Reply:
Sorry for the delay - I've been away. 

Good point! I've added these in along with some tests.